ovs-p4rt/sidecar/newovsp4rt.cc

```diff
@@ -2269,39 +2269,29 @@ void DoConfigFdbEntry(ClientInterface& client,
           GetFdbTunnelTableEntry(client, learn_info, p4info, true);
       if (status_or_read_response.ok()) {
         // Return if entry already exists.
-        return;
+        return status_or_read_response.status();
       }
     }
 
-    status =
-        ConfigFdbTunnelTableEntry(client, learn_info, p4info, insert_entry);
-    if (!status.ok()) {
-      // Ignore errors (why?)
-    }
+    // Ignore errors (why?)
+    (void)ConfigFdbTunnelTableEntry(client, learn_info, p4info, insert_entry);
```

Comment:
Is the `(void)` addition to suppress compiler warnings?

Reply:
Yes. It is also a way of letting the reader know that we _know_ the function returns a value and are choosing to ignore it.

The _comment_ is shorthand for "I really wish the original developer had either handled the returned the return or left a comment explaining why they chose to ignore it. I really wish they had done a better job, but I'm not in a position to do anything about it right now. I will, however, leave a marker in the code for the benefit of future developers."